apps/contributions/tests/test_models.py

```diff
@@ -550,16 +578,22 @@ def handle_thank_you_email_contribution(self, request):
 
     @pytest.mark.usefixtures("_mock_stripe_customer")
     @pytest.mark.parametrize("send_receipt_email_via_nre", [True, False])
-    def test_handle_thank_you_email(self, contribution, send_receipt_email_via_nre, mocker, settings):
+    @pytest.mark.parametrize("show_billing_history", [None, True])
+    def test_handle_thank_you_email(
+        self, contribution: Contribution, send_receipt_email_via_nre, show_billing_history, mocker, settings
+    ):
         """Show that when org configured to have NRE send thank you emails, send_templated_email gets called with expected args."""
         settings.CELERY_TASK_ALWAYS_EAGER = True
         (org := contribution.revenue_program.organization).send_receipt_email_via_nre = send_receipt_email_via_nre
         org.save()
         send_thank_you_email_spy = mocker.spy(send_thank_you_email, "delay")
 
         mocker.patch("apps.contributions.models.Contributor.create_magic_link", return_value="fake_magic_link")
-        contribution.handle_thank_you_email()
+        contribution.handle_thank_you_email(show_billing_history)
```

Comment:
Better to be explicit here since tests serve as documentation:

```suggestion
        contribution.handle_thank_you_email(send_billing_history=show_billing_history)
```